packages/@aws-cdk/aws-sqs/README.md

```diff
@@ -44,7 +44,8 @@ The following encryption modes are supported:
 * KMS key that you can managed yourself
 * Server-side encryption managed by SQS (SSE-SQS)
 
-Support for SSE-SQS is available in all AWS Commercial and GovCloud Regions except the China Regions.
+All newly created queues are encrypted by default with SSE-SQS since this is the new default encryption for SQS queues. Queues created in the past are unencrypted by default.
```

Comment:
@corymhall Should I also remove explanation of the old default value here? Then it's similar to the docstring. On the other hand, users could ask why their queue isn't encrypted. In this case, the explanation could help to understand the new behavior.

Reply:
I think if know the date that this behavior was enabled (was it when they released SSE-SQS?) then we can say queues created prior to DATE ... etc.

Reply:
Unfortunately, I can't find a date since when this change is active. The issue was reported 13 days ago (#22137). Therefore, I don't think that the default is active since SQS was released in November 2021. Moreover, the [announcement](https://aws.amazon.com/about-aws/whats-new/2021/11/amazon-sqs-server-side-encryption-keys-sse/) does not contain any reference to this. 

I also tried to figure out the date looking at the SQS documentation. There is a version on GitHub, unfortunately the change isn't replicated [there](https://github.com/awsdocs/amazon-sqs-developer-guide/blob/main/doc_source/sqs-configure-sse-existing-queue.md).

So how should we proceed? To you have another idea? Or should I just remove the note that explains the old default value (unencrypted)?

Reply:
I would say lets just remove any mention of the old behavior.